docs/NSR.md

```diff
@@ -17,21 +17,21 @@ a.	Segment Certificate Systems into networks based on their functional or logica
 
 b.	Apply equivalent security controls to all systems co-located in the same network with a Certificate System;
 
-c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks;
+c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks. Offline Root CA systems shall be a non-networked, stand-alone system that is powered down when not in use; 
```

Comment:
The problem I have with this OR statement is it now allows Root CAs to be online, similar to how we have Common:

Maintain Root CA Systems in a High Security Zone and in an offline state AND air-gapped from all other networks.